runtime/exporter-stdout/src/main/java/io/aklivity/zilla/runtime/exporter/stdout/internal/stream/StdoutEventsStream.java

```diff
@@ -43,14 +45,15 @@ public StdoutEventsStream(
         PrintStream out)
     {
         this.context = context;
+        this.stdoutExporterTypeId = context.supplyTypeId(StdoutExporter.NAME);
         this.readEvent = context.supplyEventReader();
         this.formatter = context.supplyEventFormatter();
         this.out = out;
     }
 
     public int process()
     {
-        return readEvent.read(this::handleEvent, 1);
+        return readEvent.read(this::handleEvent, stdoutExporterTypeId, 1);
```

Comment:
This approach breaks if we have 2 exporter instances of the same type.

Let's instead remove the complexity of creating multiple buffer spies in the events layout requiring the change to the interface, and instead create a new spy when a new event reader is supplied. That is the signal to us that we need independent spy because we have an independent event reader, which needs to keep track of its own progress through the events.